solutions/cf-609E.mdx

```diff
@@ -0,0 +1,182 @@
+---
+id: cf-609E
+source: CF
+title: Minimum spanning tree for each edge
+author: Chongtian Ma
+---
+
+[Official Analysis](https://codeforces.com/blog/entry/22187)
```

Comment:
add lang

Reply:
the editorial doesnt provide a sample code...

Reply:
oops ok mb